sandbox/public/onBeforeEventSentHelper.js

```diff
@@ -14,16 +14,17 @@ function sendAnalyticsPayload({ analyticsPayload, visitorID }) {
     });
 }
 
-function getAnalyticsPayload(proposition) {
+function getClickAnalyticsToken(proposition) {
```

Comment:
Just nitpicking and suggesting `getAnalyticsTokenForClick`, but I would also be fine if name is not changed.